client/packages/host/src/routers/ManageRouter.tsx

```diff
@@ -6,33 +6,29 @@ const NameService = React.lazy(
   () => import('@openmsupply-client/system/src/Name/Service')
 );
 
-const IndicatorsDemographicsService = React.lazy(
-  () =>
-    import(
-      '@openmsupply-client/system/src/IndicatorsDemographics/Service/Service'
-    )
+const ManageService = React.lazy(
+  () => import('@openmsupply-client/system/src/Manage/Service')
 );
 
 const fullFacilitiesPath = RouteBuilder.create(AppRoute.Manage)
   .addPart(AppRoute.Facilities)
   .addWildCard()
   .build();
 
-const fullIndicatorsDemographicsPath = RouteBuilder.create(AppRoute.Manage)
-  .addPart(AppRoute.IndicatorsDemographics)
+const fullManagePath = RouteBuilder.create(AppRoute.Manage)
   .addWildCard()
   .build();
 
 export const ManageRouter: FC = () => {
   const gotoFacilities = useMatch(fullFacilitiesPath);
-  const gotoIndicatorsDemographics = useMatch(fullIndicatorsDemographicsPath);
+  const goToManage = useMatch(fullManagePath);
 
   if (gotoFacilities) {
     return <NameService />;
   }
 
-  if (gotoIndicatorsDemographics) {
-    return <IndicatorsDemographicsService />;
+  if (goToManage) {
+    return <ManageService />;
```

Comment:
Demographics was just one page under `Manage` section. Now that we are adding a second one, thought best to refactor "indicatorservice" to be manage service, and demographics just a page within it. Sorry for extra noise, it's all just moving files accordingly 🙏